komga-webui/src/services/komga-series.service.ts

```diff
@@ -116,4 +116,8 @@ export default class KomgaSeriesService {
       throw new Error(msg)
     }
   }
+
+  public fromBook (book: BookDto): Promise<SeriesDto> {
```

Comment:
seems useless, you could just call `getOneSeries(book.seriesId)` instead of adding a function